apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/attach-launch-sidebar.tsx

```diff
@@ -35,6 +36,12 @@ registerPlugin( 'a8c-editor-site-launch', {
 			return null;
 		}
 
-		return <LaunchModal onClose={ closeSidebar } />;
+		// TODO: this flag should come from A/B testing
+		const isFocusedLaunch = true;
+		return isFocusedLaunch ? (
+			<FocusedLaunchModal onClose={ closeSidebar } />
```

Comment:
Note: The new `FocusedLaunchModal` re-uses the same state/actions as the control launch modal (i.e. `isSidebarOpen`, `closeSidebar`, ...)